docs/faq/faq.md

```diff
@@ -155,3 +155,12 @@ IP and port to indicate which customer to route traffic to.
 As with all publicly accessible IPs and ports on the internet anyone can send
 traffic to those endpoints, but ngrok provides a [number of mechanisms](https://ngrok.com/docs/guides/security-dev-productivity/) to control
```

Comment:
traffic to those endpoints, but ngrok provides a [number of mechanisms](/docs/other-guides/security-dev-productivity/) to control

Reply:
Update the link